ui/src/store/features/bunny-cdn/bunny-cdn-client.ts

```diff
@@ -0,0 +1,80 @@
+import { env } from '@/constants';
+import { AppLog } from '@/utils';
+import axios from 'axios';
+import * as crypto from 'crypto';
+
+const client = (body: string, endpoint: string) => {
+  if (!env.bunnyCDN.feSigningKey) {
+    AppLog.error('Missing BunnyCDN signing key');
+    throw new Error();

```

Comment:
should we add a message to all these errors?

Reply:
gotcha